lib/test-definition.rb

```diff
@@ -196,6 +196,7 @@ def initialize(name, &blk)
     @blk = blk
     @current_label_id = 0
     @timeout = 10
+    @repeat_on_failure = 0
```

Comment:
I think `allowed_retries` or (for video game fans!) `num_lives` would be a better name.